dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -69,6 +77,10 @@ namespace Microsoft.Windows.AppNotifications.Builder
         // Launches the URI passed into the button when activated.
         AppNotificationButton SetInvokeUri(Windows.Foundation.Uri protocolUri);
         AppNotificationButton SetInvokeUri(Windows.Foundation.Uri protocolUri, String targetAppId);
+
+        // Sets the setting type for the button.
+        [contract(AppNotificationBuilderContract, 2)]
+        AppNotificationButton SetSettingType(AppNotificationButtonSettingType value);
```

Comment:
`SetButtonStyle(...)`

Reply:
Renamed to "SetSettingStyle", Enum class : winrt::AppNotificationButtonSettingStyle